dashboard/app/models/user.rb

```diff
@@ -1474,6 +1474,7 @@ def send_reset_password_for_users(email, users)
       raw, _enc = Devise.token_generator.generate(User, :reset_password_token)
       self.child_users = unique_users
       send_devise_notification(:reset_password_instructions, raw, {to: email})
+      return self
```

Comment:
why is this necessary?

Reply:
We have to return the object that contains the `child_users` (see two lines up) in order to display the names. Previously it was returning the devise notification from the previous line.

Reply:
there it is! I was trying to find that line but missed it. SGTM